### PR TITLE
Archive - archive experiments so they don't come back from the dead [¬º-°]¬

### PR DESCRIPTION
**Pull request by spacerockzero**
See: https://cloudup.com/cSJdpIPJ-m9

NOTE: Requires the https://github.com/XPRMNTL/xpr-dash-mongodb/pull/1 PR merged as well, since there is a schema change

### CODE REVIEW

File: assets/css/feature.css [hunk 1]
```diff
@@ -141,6 +141,31 @@ a {
   padding-top: 0;
   padding-bottom: 0;
 }
+.archived {
+  padding: 1em 3em;
+  background-color: rgba(1,1,1,0.1);
```
Comment (dncrews) [inline assets/css/feature.css]: ?

File: package.json [hunk 2]
```diff
@@ -18,7 +19,7 @@
     "stylus": "^0.51.1",
     "superagent": "^1.2.0",
     "superagent-defaults": "^0.1.12",
-    "xpr-dash-mongodb": "^2.0.0",
+    "xpr-dash-mongodb": "git+https://github.com/fs-webdev/xpr-dash-mongodb.git#archive",
```
Comment (dncrews) [inline package.json]: Hold on this

**Review by dncrews — CHANGES_REQUESTED** [review]

File: assets/css/feature.styl [hunk 3]
```diff
@@ -143,6 +143,30 @@ a
   padding-top 0
   padding-bottom 0
 
+.archived
+  padding 1em 3em
+  background-color rgba(1,1,1,0.1)
```
Comment (dncrews) [inline assets/css/feature.styl]: ?

File: assets/css/feature.styl [hunk 4]
```diff
@@ -143,6 +143,30 @@ a
...
+    & > span
+      display flex
+      align-items center
+
+      & > .glyphicon
+        color #999
+        font-size 130%
+
+  & > div > .panel
+    border 2px dashed rgba(255,255,255,1)
```
Comment (dncrews) [inline assets/css/feature.styl]: ?

**Review by dncrews — CHANGES_REQUESTED** [review]

**Comment by spacerockzero** [comment]
@dncrews I've fixed the issues you mentioned.

**Review by dncrews — APPROVED** [review]

**Comment by spacerockzero** [comment]
woot!

**Pull request merged**